kaggle_environments/envs/connectx/test_connectx.py

```diff
@@ -64,7 +64,7 @@ def test_can_reset():
             "action": 0,
             "status": "ACTIVE",
             "info": {},
-            "observation": {"remainingOverageTime": 60, "board": [0] * 42, "mark": 1},
+            "observation": {"remainingOverageTime": 60, "board": [0] * 42, "mark": 1, "step": 0},
```

Comment:
Several environments didn't previously implement `observation.step` so tests have been updated to reflect.